include/openthread/link.h

```diff
@@ -1121,12 +1143,32 @@ otError otLinkSetEnabled(otInstance *aInstance, bool aEnable);
  *
  * @param[in]  aInstance A pointer to an OpenThread instance.
  *
- * @retval true   Link layer is enabled.
- * @retval false  Link layer is not enabled.
+ * @retval TRUE   Link layer is enabled.
+ * @retval FALSE  Link layer is not enabled.
  *
  */
 bool otLinkIsEnabled(otInstance *aInstance);
 
+/**
+ * This function indicates whether or not the link layer is csl enabled.
```

Comment:
Is it more accurate to say "...whether or not the link layer **has** CSL enabled."?

Reply:
Or simply "...whether or not CSL is enabled."?

Reply:
> Or simply "...whether or not CSL is enabled."?

SGTM!

Reply:
> Or simply "...whether or not CSL is enabled."?

Yes let's do that, please